core/src/main/java/org/mule/runtime/core/internal/event/AbstractEventContext.java

```diff
@@ -168,19 +168,19 @@ public final void success(CoreEvent event) {
   @Override
   public final Publisher<Void> error(Throwable throwable) {
     if (isResponseDone()) {
-      if (LOGGER.isDebugEnabled()) {
-        LOGGER.debug(this + " error response was already completed, ignoring.");
+      if (debugLogEnabled) {
+        LOGGER.debug("{} error response was already completed, ignoring.", this);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: No need to call "toString()" method as formatting and string conversion is done by the Formatter. ([squid:S3457](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3457))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7701&issues=AWkLjRWXotIXnmsUmgk-&open=AWkLjRWXotIXnmsUmgk-)

[ISSUE_KEY]: AWkLjRWXotIXnmsUmgk-